src/components/tx/TxData.js

```diff
@@ -182,7 +192,9 @@ class TxData extends React.Component {
    * @param {Object} e Event emitted when clicking link
    */
   toggleFeatureActivation = async e => {
-    e.preventDefault();
+    if (this.props.newUiEnabled === false) {
```

Comment:
issue: Now the same problem that was happening with the "Show Raw" feature is happening here.

question: Why is there a conditional here for this `preventDefault()`? Shouldn't this command always be run for this event on any UI?

Reply:
Fixed conditional. We created this conditional because in the old UI the toggleFeatureActivation is used with an <a>, and in the new one it is only used to toggle the state of the dropdown menu, without navigation, so it is not necessary to use e.preventDefault(). 🙂